docs/agent-guide.md

```
@@ -69,11 +69,11 @@ Skills and workflows are not slash commands or buttons — you invoke them throu
 → Claude loads triage-bug
 ```
 
-**Name the workflow directly** when you want the full chain:
+**Name the agent directly** when you want the full chain:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)